src/app/components/charts/plotly-chart.ts

```diff
@@ -7,6 +7,8 @@ declare const window: any;
 @Component({
   selector: 'budgetkey-chart-plotly',
   template: `
+    <p *ngIf="firstYear < lastYear" > תקציב זה פעיל בין השנים : {{firstYear}} - {{ lastYear }}  </p>
```

Comment:
This is not a good place for this change - it's not part of this generic, reusable chart but should only be part of the budget pages.